satellite/discovery/service.go

```diff
@@ -38,38 +36,33 @@ type Config struct {
 //
 // architecture: Chore
 type Discovery struct {
-	log   *zap.Logger
-	cache *overlay.Service
-	kad   *kademlia.Kademlia
+	log     *zap.Logger
+	cache   *overlay.Service
+	contact *contact.Service
 
 	refreshLimit       int
 	refreshConcurrency int
 
-	Refresh   sync2.Cycle
-	Discovery sync2.Cycle
```

Comment:
do we need to remove the `DiscoveryInterval` config setting as well?

Reply:
yes good catch!